lib/routes/objects.js

```diff
@@ -230,6 +230,18 @@ module.exports =
             expiration
           });
 
+          if (fastify.log.security) {
+            fastify.log.security({
+              success: true,
+              message: 'Object variant sucessfully created',
+              method: req.method,
+              url: req.url,
```

Comment:
Similarly here, is method + url enough to know what was created?

Reply:
Same as above.